provisioners/apt_proxy.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+if [ "${apt_cache_host:-}" ]
```

Comment:
should this be `if [ -n "${apt_cache_host:-}" ]`?

Reply:
`-n` is the default test. but specifing `-n` is more explicit.